src/pencilkit.cs

```diff
@@ -70,6 +86,15 @@ enum PKCanvasViewDrawingPolicy : ulong {
 		PencilOnly,
 	}
 
+	[iOS (17, 0), Mac (14, 0)]
+	[Introduced (PlatformName.MacCatalyst, 17, 0)]
+	[Native]
+	public enum PKContentVersion : long {
+		Version1 = 1,
+		Version2 = 2,
+		Latest = Version2,
```

Comment:
We can't actually do a "Latest", because if a new version is added, then we'll end up with a breaking change.

In the past we've just not bound these non-constant enum values.

```suggestion
```